src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1470,7 +1470,7 @@ def create_backup(cmd, resource_group_name, webapp_name, storage_account_url,
     if backup_name and backup_name.lower().endswith('.zip'):
         backup_name = backup_name[:-4]
     db_setting = _create_db_setting(cmd, db_name, db_type=db_type, db_connection_string=db_connection_string)
-    backup_request = BackupRequest(backup_request_name=backup_name,
+    backup_request = BackupRequest(backup_name=backup_name,
```

Comment:
Why you change this line? Is it a mistake before? or SDK has a change?

Reply:
Yes, there is a SDK change.